rootfs/Dockerfile

```diff
@@ -1,41 +1,24 @@
-FROM quay.io/deis/base:v0.3.5
+FROM registry:2.5.1
 
-COPY . /
+RUN apk add --no-cache \
+        gcc \
+        git \
+        linux-headers \
```

Comment:
Can these development-related `apk` packages be purged later? I assume they are just present to support `pip install`.